proposals/description/fip_82.ts

```diff
@@ -0,0 +1,144 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_82: ProposalDescription = {
+  title: 'FIP-82: Deploy TribalCouncil',
+  commands: [
+    //////////// GRANT POD TIMELOCKS RELEVANT ACCESS ROLES ///////////
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0x2172861495e7b85edac73e3cd5fbb42dd675baadf627720e687bcfdaca025096', // ROLE_ADMIN
+        '0x899bd46557473cb80307a9dabc297131ced39608330a2d29b2d52b660c03923e' // GOVERN_ROLE
+      ],
+      description: 'Create ROLE_ADMIN role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0xa8d944a5277d6a203f114d020d26918a390f167b089a46be4fca9da716d23783', // ORACLE_ADMIN
+        '0x899bd46557473cb80307a9dabc297131ced39608330a2d29b2d52b660c03923e' // GOVERN_ROLE
+      ],
+      description: 'Create ORACLE_ADMIN role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: ['0x2172861495e7b85edac73e3cd5fbb42dd675baadf627720e687bcfdaca025096', '{tribalCouncilTimelock}'],
+      description: 'Grant Tribal Council ROLE_ADMIN'
```

Comment:
what does ROLE_ADMIN do

Reply:
I've polluted this PR slightly with the roles PR I'm submitting. `ROLE_ADMIN` is the role for the `TribalCouncil`. 

I want to be able to allow this role to grant a subset of other lesser roles so that the TribalCouncil can authorise child pods